packages/admin/cms-admin/src/form/file/FinalFormFileUpload.tsx

```diff
@@ -119,12 +125,18 @@ export const FinalFormFileUpload = <Multiple extends boolean | undefined>({
                             onChange([...inputValue, ...successfullyUploadedFiles]);
                         }
                     } else {
+                        let errorMessage = <FormattedMessage id="comet.finalFormFileUpload.uploadFailed" defaultMessage="Upload failed." />;
```

Comment:
nit: Could we also add this to the commonErrorMessages?

Reply:
We could but since it's only used once, it's not really "common" 🤷🏼‍♂️